go-app-ussd_higherhealth_healthcheck.js

```diff
@@ -1834,6 +1834,37 @@ go.app = (function () {
       });
     });
 
+    self.add("state_start_triage", function (name, opts) {
+      if (!self.im.config.study_b_enabled) {
+        return self.states.create("state_age");
+      }
+
+      if (self.im.user.answers.hc_start_timestamp === undefined) {
+        var msisdn = utils.normalize_msisdn(self.im.user.addr, "ZA");
+        return new JsonApi(self.im).post(
+          self.im.config.eventstore.url + "/api/v2/covid19triagestart/", {
+            headers: {
+              "Authorization": ["Token " + self.im.config.eventstore.token],
+              "User-Agent": ["Jsbox/HH-Covid19-Triage-USSD"]
+            },
+            data: {"msisdn": msisdn, "source": "USSD"}
+          }).then(function (response) {
+            self.im.user.answers.hc_start_timestamp = response.data.timestamp;
+            return self.states.create("state_age");
+          }, function (e) {
+            // Go to error state after 3 failed HTTP requests
+            opts.http_error_count = _.get(opts, "http_error_count", 0) + 1;
+            console.log("err count:", opts.http_error_count);
```

Comment:
`console.log` actually breaks the sandbox, even though the tests run fine, because `console` isn't allowed to be in the scope as part of the sandboxing. There's a `self.im.log` that we can use instead.

Reply:
Oops, that's debug stuff that I forgot to remove.